di-ipv-orchestrator-stub/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gradle:7.2.0-jdk17@sha256:0619ec0c28cdd0f5f7ba389e7fea845b400cf282dd28b46b4d67f60a009732fa AS build
+FROM gradle:7.3.0-jdk17@sha256:1f65f4991aa496cf835cbce10290bdc71c1cb526b4df807d97c4f51b863f2399 AS build
```

Comment:
An alternative way of doing this would be to just use a JDK image, and then use `./gradlew`, which means we don't need to tie our Gradle version to the version in the Wrapper - thoughts?

Reply:
Yes, that might be a good way of doing it.